apps/explorer/src/app/components/proposals/proposals-table.tsx

```diff
@@ -81,27 +87,16 @@ export const ProposalsTable = ({ data }: ProposalsTableProps) => {
         maxWidth: 150,
         hide: window.innerWidth <= BREAKPOINT_MD,
         headerName: t('Closing date'),
-        field: 'terms.closingDatetime',
-        valueFormatter: ({
-          value,
-        }: VegaValueFormatterParams<
-          ProposalListFieldsFragment,
-          'terms.closingDatetime'
-        >) => {
-          return value ? getDateTimeFormat().format(new Date(value)) : '-';
+        valueGetter: ({ data }) => {
+          return (
+            data?.terms?.closingDatetime || data.batchTerms.closingDatetime
+          );
         },
-      },
-      {
-        colId: 'eDate',
-        maxWidth: 150,
-        hide: window.innerWidth <= BREAKPOINT_MD,
-        headerName: t('Enactment date'),
-        field: 'terms.enactmentDatetime',
         valueFormatter: ({
           value,
         }: VegaValueFormatterParams<
           ProposalListFieldsFragment,
-          'terms.enactmentDatetime'
+          'terms.closingDatetime'
```

Comment:
Note: this diff is a little misleading, I definitely didn't just change the enactment column to show closing